target/linux/ramips/image/mt7621.mk

```diff
@@ -641,6 +649,31 @@ define Device/cudy_x6-v2
 endef
 TARGET_DEVICES += cudy_x6-v2
 
+define Device/dlink_covr-x1860-a1
+  $(Device/dsa-migration)
```

Comment:
Is this needed since the device didn't exists before?

Reply:
apparently not, it was mostly a boilerplate found with the majority of the other devices (but indeed it just declares the COMPAT version, so could most probably be dropped)

Reply:
I removed the line